packages/picker/src/Picker.ts

```diff
@@ -319,7 +319,7 @@ export class PickerBase extends SizedMixin(Focusable) {
                     selected?: boolean;
                 }
             ) => {
-                if (this.value === el.value) {
+                if (this.selects != null && this.value === el.value) {
```

Comment:
Does the amount to the same as testing for truthiness? A `!=` always feels surprising, so it might be good just to add a quick comment for the future. Applies to the below usage as well.

Reply:
`== null` tests for `null | undefined` (https://stackabuse.com/javascript-check-if-variable-is-a-undefined-or-null/)

It's a pattern I picked up on my last team where it also stood out to me because of the two-char check.

I encapsulated this check everywhere selected is set, & added a comment there.